webshare/share-non-string-manual.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>WebShare Manual Test: Share non-string types (test implicit conversion)</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="resources/manual-helper.js"></script>
+  </head>
+  <body>
+    <script>
+        setup({explicit_timeout: true});
+
+        // Expect that each of the non-string values is converted into a string.
```

Comment:
Done.

I just moved things around so we test the integer with url. But I also took your idea to have an object with `toString` to test that the `toString` method is actually called, instead of just converting objects into the default string.